### PR TITLE
Update videos.md

### PR DESCRIPTION
**Pull request by sewhite**
The videos have been split into topics and edited. The list of links has been updated to reflect that.

### CODE REVIEW

File: developer/videos.md [hunk 1]
```diff
@@ -4,13 +4,46 @@ title: Keyman Developer Training Videos
...
+
+[Creating the keyboard package](https://vimeo.com/398939211) 42:08
+
+[Creating the predictive text model](https://vimeo.com/399132672) 44:36
+
+* Day 5 - Promotion and wrapup
+
+[Promoting your keyboard](https://vimeo.com/399162811)  20:01
+
+[Keyman Community and Documentation](https://vimeo.com/399264352) 13:50
```
Comment (darcywong00) [inline developer/videos.md]: This link is https://vimeo.com/399237808

File: developer/videos.md [hunk 2]
```diff
@@ -4,13 +4,46 @@ title: Keyman Developer Training Videos
 
 ## Keyman Developer Online Workshop Videos (Feb 2020)
 
-These five videos comprise the seminar sessions of our Online Workshop held in February 2020.
+Our seminar sessions during the five day workshop were recorded in these videos:
+
+* Day 1 - Introduction and principles of keyboard design
+
+[Keyman MultiPlatform Keyboards](https://vimeo.com/398526050)  32:38
```
Comment (darcywong00) [inline developer/videos.md]: Thanks for splitting up the content.
I suggest indenting each video link with 4 spaces

**Comment by mcdurdin** [comment]
Amusing -- I didn't see this before I did the same kinda think at #161! Whichever you think works better, @darcywong00!

**Comment by darcywong00** [comment]
Thanks for the PR @sewhite . I think we'll go with #161 because it offers videos by topic and by day (2 lists to choose).

**Pull request closed without merging**